packages/composition/dbaas/exoscale/opensearch.yml

```diff
@@ -8,17 +8,16 @@ parameters:
 
   appcat:
     compositions:
-      "exoscale.opensearch.appcat.vshn.io":
+      "opensearch.exoscale.appcat.vshn.io":
```

Comment:
For consistency this should be `exoscaleopensearch.exoscale...`.

Reply:
Fixed